bugs/github-10430/test-results.md

```diff
@@ -0,0 +1,326 @@
+^C2022-01-12T20:44:24.894674Z  INFO vector: Vector has stopped.
+2022-01-12T20:44:24.894731Z  INFO source{component_kind="source" component_id=stdin component_type=stdin component_name=stdin}: vector::sources::stdin: Finished sending.
+2022-01-12T20:44:24.895894Z  INFO vector::topology::running: Shutting down... Waiting on running components. remaining_components="http_tarpit" time_remaining="59 seconds left"
+toby@consigliere:~/src/vector/bugs/github-10430$ ls -l /tmp/vector/github-10430/
+total 4
+drwxr-xr-x 2 toby toby 4096 Jan 12 15:44 http_tarpit_id
+toby@consigliere:~/src/vector/bugs/github-10430$ ./vector-v0.18.1 -c config.toml
+2022-01-12T20:44:46.335761Z  INFO vector::app: Log level is enabled. level="vector=info,codec=info,vrl=info,file_source=info,tower_limit=trace,rdkafka=info"
+2022-01-12T20:44:46.335805Z  INFO vector::app: Loading configs. paths=["config.toml"]
+2022-01-12T20:44:46.337708Z  INFO vector::sources::stdin: Capturing STDIN.
+2022-01-12T20:44:46.370628Z  INFO vector::topology::running: Running healthchecks.
+2022-01-12T20:44:46.370658Z  INFO vector::topology::running: Starting source. key=stdin
+2022-01-12T20:44:46.370688Z  INFO vector::topology::running: Starting sink. key=http_tarpit
+2022-01-12T20:44:46.370684Z  INFO vector::topology::builder: Healthcheck: Passed.
+2022-01-12T20:44:46.370723Z  INFO vector: Vector has started. debug="false" version="0.18.1" arch="x86_64" build_id="c4adb60 2021-11-30"
+2022-01-12T20:44:46.370732Z  INFO vector::app: API is disabled, enable by setting `api.enabled` to `true` and use commands like `vector top`.
+^C2022-01-12T20:44:47.972404Z  INFO vector: Vector has stopped.
+2022-01-12T20:44:47.972452Z  INFO source{component_kind="source" component_id=stdin component_type=stdin component_name=stdin}: vector::sources::stdin: Finished sending.
+2022-01-12T20:44:47.973543Z  INFO vector::topology::running: Shutting down... Waiting on running components. remaining_components="http_tarpit" time_remaining="59 seconds left"
+toby@consigliere:~/src/vector/bugs/github-10430$ ls -l /tmp/vector/github-10430/
+total 8
+drwxr-xr-x 2 toby toby 4096 Jan 12 15:44 http_tarpit_buffer
+drwxr-xr-x 2 toby toby 4096 Jan 12 15:44 http_tarpit_id
+toby@consigliere:~/src/vector/bugs/github-10430$ ./vector-fix -c config.toml
+2022-01-12T20:45:03.147077Z  INFO vector::app: Log level is enabled. level="vector=info,codec=info,vrl=info,file_source=info,tower_limit=trace,rdkafka=info,buffers=info"
+2022-01-12T20:45:03.147126Z  INFO vector::app: Loading configs. paths=["config.toml"]
+2022-01-12T20:45:03.147953Z  INFO vector::sources::stdin: Capturing STDIN.
+2022-01-12T20:45:03.185627Z  INFO vector::topology::running: Running healthchecks.
+2022-01-12T20:45:03.185664Z  INFO vector::topology::running: Starting source. key=stdin
+2022-01-12T20:45:03.185688Z  INFO vector::topology::running: Starting sink. key=http_tarpit
```

Comment:
Should we have a log about the old buffer being renamed?

Reply:
Good idea: added a log messaging to that effect.